machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -56,7 +56,7 @@ The status of the control plane machine set after installation depends on your c
 [.small]
 --
 1. AWS clusters that are upgraded from version 4.11 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
-2. GCP and Azure clusters that are upgraded from version 4.12 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
+2. GCP, Azure, and Nutanix clusters that are upgraded from version 4.12 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
```

Comment:
Perhaps a new line should be added for Nutanix, because Nutanix clusters that are upgraded from version **4.13** also require activating cpms.

Reply:
Ah, I misunderstood the version difference. Good idea, I will break it out separately for clarity on that point :)